di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/models/JarUserInfo.java

```diff
@@ -8,10 +8,16 @@ public record JarUserInfo(
                 Essential coreIdentityJwtClaim,
         @JsonProperty(value = "https://vocab.account.gov.uk/v1/address") Essential addressClaim,
         @JsonProperty(value = "https://vocab.account.gov.uk/v1/passport") Essential passportClaim,
-        @JsonProperty(value = "https://vocab.account.gov.uk/v1/socialSecurityNumber")
+        @JsonProperty(value = "https://vocab.account.gov.uk/v1/socialSecurityRecord")
                 Essential ninoClaim,
         @JsonInclude(JsonInclude.Include.NON_NULL)
                 @JsonProperty(value = "https://vocab.account.gov.uk/v1/inheritedIdentityJWT")
                 ListOfStringValues inheritedIdentityClaim,
         @JsonProperty(value = "https://vocab.account.gov.uk/v1/storageAccessToken")
-                ListOfStringValues evcsAccessToken) {}
+                ListOfStringValues evcsAccessToken,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+                @JsonProperty(value = "https://vocab.account.gov.uk/v1/drivingPermit")
+                Essential drivingPermitClaim,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+                @JsonProperty(value = "https://vocab.account.gov.uk/v1/returnCode")
+                Essential returnCodeClaim) {}
```

Comment:
Can we put this next to the other requested identity claims (i.e. alongside passport etc.) so they're separate from the weird ones (inherited identity and storage access token).